Investigation/go/task_queue_demo/sendtask.go

```diff
@@ -0,0 +1,10 @@
+package main
```

Comment:
How about deploying this demo as well as Redis on which this demo relies to a Minikube Kubernetes cluster?